units/commit_queue.go

```diff
@@ -74,12 +74,12 @@ func (j *commitQueueJob) TryUnstick(cq *commitqueue.CommitQueue) {
 	}
 
 	// unstuck the queue if the patch is done.
-	if !patch.IsValidId(nextItem.Issue) {
+	if nextItem.Source == commitqueue.SourceDiff && !patch.IsValidId(nextItem.Issue) {
 		j.dequeue(cq, nextItem)
 		j.logError(errors.Errorf("The Patch id '%s' is not an object id", nextItem.Issue), "The patch was removed from the queue.", nextItem)
 		return
 	}
-	patchDoc, err := patch.FindOne(patch.ByStringId(nextItem.Issue).WithFields(patch.FinishTimeKey, patch.StatusKey))
+	patchDoc, err := patch.FindOne(patch.ByStringId(nextItem.Version).WithFields(patch.FinishTimeKey, patch.StatusKey))
```

Comment:
We don't populate Version for diffs, unless you modified that.

Reply:
Also, do commit queue items have patches? Or just the version document?

Reply:
If we're dequeuing a commit queue PR item, should we send that update to Github?

Reply:
I changed it to populate the version when the patch is finalized. Changing both at once is technically not backwards-compatible, but I'm going to do it hoping that there aren't commit queue items stuck from before deploying this that need to be unstuck after

Reply:
Okay sounds good; still a question about needing to send an update to the commit queue PR if we dequeue in this function.

Reply:
I added it a couple lines below

Reply:
oh gotcha didn't see the update